protos/flyteidl/admin/node_execution.proto

```diff
@@ -184,4 +181,7 @@ message NodeExecutionGetDataResponse {
 
     // Optional, full_outputs will only be populated if they are under a configured size threshold. 
     core.LiteralMap full_outputs               = 4;
+
+    // In the case this node yielded a task which launched a dynamic workflow, we return its structure here.
```

Comment:
```suggestion
    // Optional Workflow closure for a dynamically generated workflow, in the case this node yields a dynamic workflow we return its structure here.
```

Question, for retries this will be updated?

Reply:
done. unfortunately no, this is because the task that yields the workflow will be retried rather than the node itself.